portfolio/src/main/webapp/map-script.js

```diff
@@ -49,59 +49,59 @@ function initMap() {
                     featureType: 'administrative.locality',
                     elementType: 'labels.text.fill',
                     stylers: [{color: '#d59563'}]
```

Comment:
Missing ',' at the end? Same for blocks below.

Reply:
Ah, please ignore this one. I was writing Go at the same time and got confused..